src/Extensions/DataObjectExtension.php

```diff
@@ -90,7 +90,12 @@ protected function pushToSolr(DataObject $owner)
         try {
             Versioned::set_reading_mode(Versioned::DEFAULT_MODE);
             $service->setInDebugMode(false);
-            $service->updateItems(ArrayList::create([$owner]), SolrCoreService::UPDATE_TYPE);
+            $type = SolrCoreService::UPDATE_TYPE;
+            // If the object should show in search, remove it
```

Comment:
`// If the object should NOT show in search...`

Reply:
Blep, you're correct. Thinking is hard when on painkillers and all.